main/tests/test_models.py

```diff
@@ -27,13 +27,32 @@ def test_post_publish_method(self):
         """Publish method working ok."""
         self.test_post.publish()
         self.assertLess(self.test_post.published_date, datetime.now())
+        self.assertEqual(self.test_post.published_date, datetime(day=1, month=4, year=2016))
+        self.assertLess(datetime(day=1, month=4, year=2015), self.test_post.published_date)
+
+    def tearDown(self):
+        """Clean data for new test."""
+        del self.user
+        del self.test_post
+
+
+class ModelTestComment(TestCase):
+    """Main class for testing comment models of this project."""
+
+    def setUp(self):
+        """Prepare data for testing."""
+        self.user = User.objects.create(username='testuser')
+        self.test_post = Post.objects.create(author=self.user, title='Test', text='superText',
+                                             created_date=timezone.now())
+        self.comment = Comment.objects.create(post=self.test_post, author=self.user, text='superComment',
```

Comment:
It's worth omitting arguments which have default values set.
Also you forgot to:
* [ ] test `approve()` method